Cargo.toml

```diff
@@ -25,7 +25,7 @@ scale = { package = "parity-scale-codec", version = "3", default-features = fals
 [features]
 default = ["std"]
 std = [
-    "bitvec/std",
+    "bitvec?/std",
```

Comment:
The bitvec crate will no longer be pulled in when "std" is enabled, but this I think makes no difference to the external interface since bitvec additions are behind the separate "bit-vec" feature anyway.

Reply:
Ooh ok; I'll do a quick follow PR there then to prevent that :)

Reply:
PR for parity-scale-codec thing: https://github.com/paritytech/parity-scale-codec/pull/375